src/reducers/dependencies.reducer.js

```diff
@@ -1,7 +1,10 @@
 // @flow
 import produce from 'immer';
+import { combineReducers } from 'redux';
 
 import {
+  LOAD_DEPENDENCY_INFO_FROM_DISK_START,
+  LOAD_DEPENDENCY_INFO_FROM_DISK_ERROR,
   LOAD_DEPENDENCY_INFO_FROM_DISK_FINISH,
   ADD_DEPENDENCY,
   UPDATE_DEPENDENCY,
```

Comment:
Yeah, I know. I've kept it because I don't like to use `any` for the action type.
But I don't know why it's not working with the type. Do you have an idea? I don't remember the type error right now. But the only fix was to use `any`.